recipes/arcp/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "arcp" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "ad9df671a6cacfb44e85cd2971eee87e663a5bbf84618b31476b9b452bb01a41" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stain/arcp-py/releases/download/0.2.0/arcp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

As it's only used once there is no need to use a variable:
```suggestion
  sha256: ad9df671a6cacfb44e85cd2971eee87e663a5bbf84618b31476b9b452bb01a41
```